corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -231,6 +251,10 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
                     self.ui.selectRow.prop("checked", action === constants.MULTI_SELECT_ADD);
                 }
             });
+            self.smallScreenListener = cloudcareUtils.smallScreenListener(smallScreenEnabled => {
```

Comment:
I think this will create a new jQuery listener for every case. Is it possible to access the `smallScreenEnabled` property from the parent `CaseListView`?

Reply:
That's a valid point, but does that mean tiles will still be collapsible if the window used to be small but now it's larger? Or will it just keep the appropriate "collapsed" classes for _if_ the screen is small again? I would probably check with AEs for thoughts on design/user interaction and what behavior is most predictable.

Reply:
If the screen was small and is now larger (without reloading the page) the tiles would still be collapsible and keep the collapsed/expanded state they were in. The other way around if the window was large and is now smaller the tiles would not be collapsible and stay the expanded.

Reply:
Thinking back to the main use cases, the main time someone would change window sizes would be if they're switching devices (laptop -> tablet, tablet-> phone, phone -> laptop, etc), which would mean a page reload anyways and they wouldn't run into this situation. 

The only other time this situation may come up is if someone is on a laptop or tablet and resizes the screen, maybe because they want to look at something else at the same time? I'm okay with leaving this as-is (where the collapsible behavior doesn't necessarily change if the window size changes past a certain point), since having it change and lose what tiles were collapsed/expanded would seem like a more confusing user experience for me.